src/stdatm/tests/test_atmosphere_partials.py

```diff
@@ -0,0 +1,255 @@
+# This file is part of FAST-OAD_CS23-HE : A framework for rapid Overall Aircraft Design of Hybrid
+# Electric Aircraft.
+# Copyright (C) 2022 ISAE-SUPAERO
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+from scipy.constants import foot
+
+from ..atmosphere_partials import AtmospherePartials
+from ..atmosphere import Atmosphere
+
+STEP = 1e-6
+
+
+@pytest.fixture(scope="session")
+def altitude():
+    return np.linspace(0.0, 20000.0, int(1e6))
+
+
+def get_atmosphere(altitude, altitude_in_feet):
+    atm_part = AtmospherePartials(altitude, 0.0, altitude_in_feet)
+    return atm_part
+
+
+def get_atmosphere_minus_step(altitude):
+    atm = Atmosphere(altitude - STEP, altitude_in_feet=False)
+    return atm
+
+
+def get_atmosphere_plus_step(altitude):
+    atm = Atmosphere(altitude + STEP, altitude_in_feet=False)
+    return atm
+
+
+def get_atmosphere_minus_step_ft(altitude):
+    atm = Atmosphere(altitude / foot - STEP, altitude_in_feet=True)
+    return atm
+
+
+def get_atmosphere_plus_step_ft(altitude):
+    atm = Atmosphere(altitude / foot + STEP, altitude_in_feet=True)
+    return atm
```

Comment:
We don't need to bother for performance here, so `gettatr()`can be used to get a generic function. Therefore I suggest:
```suggestion
def get_fd_partial(altitude, parameter_name, altitude_in_feet=False):
    atm_minus_step = Atmosphere(altitude - STEP, altitude_in_feet=altitude_in_feet)
    atm_plus_step = Atmosphere(altitude + STEP, altitude_in_feet=altitude_in_feet)
    return (getattr(atm_plus_step, parameter_name) - getattr(atm_minus_step, parameter_name)) / (
        2.0 * STEP
    )

```

Reply:
Please note that I kept `STEP` as a constant, but it would be now used in only this function. It could be an internal variable, or a parameter with the default value you want.